x-pack/plugins/cases/common/api/cases/user_actions/comment.ts

```diff
@@ -6,15 +6,26 @@
  */
 
 import * as rt from 'io-ts';
-import { CommentRequestRt } from '../comment';
+import { CommentRequestRt, CommentRequestWithoutRefsRt } from '../comment';
 import type { UserActionWithAttributes } from './common';
 import { ActionTypes } from './common';
 
 export const CommentUserActionPayloadRt = rt.strict({ comment: CommentRequestRt });
+export const CommentUserActionPayloadWithoutIdsRt = rt.strict({
+  comment: CommentRequestWithoutRefsRt,
+});
 
 export const CommentUserActionRt = rt.strict({
   type: rt.literal(ActionTypes.comment),
   payload: CommentUserActionPayloadRt,
 });
 
+export const CommentUserActionWithoutIdsRt = rt.strict({
```

Comment:
I needed this type so I can validate the attributes passed to the so client when you creating a user action. The attributes do not contain any ids as they are in the references array.